e2e-coho-tests/src/test/java/tests/AdjustVolumeTest.java

```diff
@@ -0,0 +1,45 @@
+package Tests;
+
+import Pages.*;
+import Tests.AbstractBaseTests.TestBase;
+import cucumber.api.CucumberOptions;
+import cucumber.api.java.en.Given;
+import org.testng.annotations.BeforeTest;
+
+@CucumberOptions(
+        strict = true,
+        monochrome = true,
+        features = "classpath:SmartWatchSettingsTest",
+        plugin = {"pretty"}
+)
+
+public class AdjustVolumeTest extends TestBase {
```

Comment:
- would you mind to give naming for classes with steps like *Steps instead of *Test?
- would be great to group the steps by functionality, like `steps` -> `login`, `volume`, `registering` etc

Reply:
I thought of doing the same changes in my next commit. It's fine I will do the changes now